src/main/java/ch/bfh/ti/i4mi/mag/mhd/Utils.java

```diff
@@ -141,4 +141,12 @@ public static Processor retrievedDocumentSetToHttResponse() {
         };
     }
    */
+
+    public static boolean isUnprefixedOid(final String oid) {
+        return oid != null && oid.matches("\\d+(\\.\\d+)+");
```

Comment:
here and in `isUprefixedUuid`: probably it would make sense to introduce static constants in order to avoid pattern recompilation on each call.  For example,

`private static final Pattern OID_PATTERN = Pattern.compile("\\d+(\\.\\d+)+");`
``
`public static boolean isUnprefixedOid(final String oid) {`
`    return (oid != null) && OID_PATTERN.matcher(oid).matches();`
`}`

Reply:
Sure!